src/pages/organize/[orgId]/campaigns/[campId]/calendar/index.tsx

```diff
@@ -60,62 +59,15 @@ type OrganizeCalendarPageProps = {
     const eventsQuery = useQuery(['campaignEvents', orgId, campId], getCampaignEvents(orgId, campId));
     const campaignQuery = useQuery(['campaign', orgId, campId], getCampaign(orgId, campId));
+    const tasksQuery = useQuery(['tasks', orgId, campId], getCampaignTasks(orgId, campId));
     const events = eventsQuery.data || [];
+    const tasks = tasksQuery.data || [];
     const campaigns = campaignQuery.data ? [campaignQuery.data] : [];
-    const intl = useIntl();
-    const [calendarView, setCalendarView] = useState('week');
-    const { focusDate, setFocusDate } = useFocusDate();
-
-    const items = [
-        { id: 'week', name: intl.formatMessage({ id: 'misc.calendar.week' }) },
-        { id: 'month', name: intl.formatMessage({ id: 'misc.calendar.month' }) },
-    ];
-
-    const handleChange = (event: React.ChangeEvent<{ name?: string; value: unknown}>) => {
-        setCalendarView(event.target.value as string);
-    };
 
     return (
         <>
-            <Box height={ 1 } position="relative">
-                <Box display="flex" justifyContent="flex-end" mr={ 4 } position="absolute" right={ 0 } top={ 1 } zIndex={ 12 }>
-                    <FormControl
-                        aria-label={ intl.formatMessage(
-                            { id: 'misc.calendar.label' }) }
-                        variant="outlined">
-                        <Select id="demo-simple-select-outlined"
-                            labelId="demo-simple-select-outlined-label"
-                            onChange={ handleChange }
-                            style={{ height: '2rem', padding: 0 }}
-                            value={ calendarView }>
-                            { items.map(item => (
-                                <MenuItem key={ item.id } value={ item.id }>
-                                    { item.name }
-                                </MenuItem>
-                            )) }
-                        </Select>
-                    </FormControl>
-                </Box>
-                <Box height={ 1 }>
-                    { calendarView === 'month' &&
-                    <MonthCalendar
-                        baseHref={ `/organize/${orgId}/campaigns/${campId}` }
-                        campaigns={ campaigns }
-                        events={ events }
-                        focusDate={ focusDate }
-                        onFocusDate={ date => setFocusDate(date) }
-                        orgId={ orgId }
-                    /> }
-                    { calendarView === 'week' &&
-                    <WeekCalendar
-                        baseHref={ `/organize/${orgId}/campaigns/${campId}` }
-                        campaigns={ campaigns }
-                        events={ events }
-                        focusDate={ focusDate }
-                        onFocusDate={ date => setFocusDate(date) }
-                        orgId={ orgId }
-                    /> }
-                </Box>
+            <Box height={ 1 }>
```

Comment:
What does this box do?

Reply:
Same applies to the other page this is used :+1:

Reply:
apparently nothing, I'll remove